docs/reference/commandline/swarm_join_token.md

```diff
@@ -65,6 +66,12 @@ SWMTKN-1-3pu6hszjas19xyp7ghgosyx9k8atbfcr8p2is99znpy26u2lkl-b30ljddcqhef9b9v4rs7
 ### `--rotate`
 
```

Comment:
Replace the sentence below with:

`If a token gets checked-in to version control, stolen, or a node gets compromised, you should rotate your swarm's join token. The `--rotate` flag does exactly that, it rotates and prints out the newly generated join token`